addons/resource.language.en_gb/resources/strings.po

```diff
@@ -14663,7 +14663,55 @@ msgctxt "#24151"
 msgid "Add-on migration in progress - please wait..."
 msgstr ""
 
-#empty strings from id 24152 to 24990
+#. Localized addon database name for splash screen
+#: xbmc/addons/AddonDatabase.h
+msgctxt "#24152"
+msgid "Addon"
```

Comment:
You might as well repeat the complete sentence each time instead of stitching the strings together.